README.md

```diff
@@ -88,6 +88,16 @@ Installation can be done in one of 2 options:
 
 
 ## Changelog
+### v0.9.3
+- tbtadm: added `peers` command and xdomain now shown in topology output
+- tbtadm: added `add` comand for adding to ACL without `approve`
+- tbtadm: fixed adding to ACL in secure level 0
```

Comment:
security level

Reply:
2. Fixes should be under `resolved issues` (or, if you prefer, merge the sections)

Reply:
@ybernat Thanks for review, I would prefer to squash them to Changelog for easy navigation, I will fix those issues and add squashing patch.